fe/fe-core/src/main/java/org/apache/doris/catalog/AzureResource.java

```diff
@@ -78,21 +81,56 @@ protected void setProperties(Map<String, String> newProperties) throws DdlExcept
         this.properties = newProperties;
     }
 
-    private static void pingAzure(String bucketName, String rootPath,
+    protected static void pingAzure(String bucketName, String rootPath,
             Map<String, String> newProperties) throws DdlException {
-        if (FeConstants.runningUnitTest) {
-            return;
+
+        String testFile = "azure://" + bucketName + "/" + rootPath + "/"
+                + UUID.randomUUID().toString() + "/test-object-valid.txt";
```

Comment:
use timestamp in milliseconds instead
e.g. doris-test-object-valid-${timstamp}.txt